elpy.el

```diff
@@ -3633,6 +3633,12 @@ or unless NAME is no callable instance."
      (require 'eldoc)
      (setq eldoc-minor-mode-string nil))
     (`buffer-init
+     ;; avoid interferences between eldoc and company meta frontend
+     (when (member 'elpy-module-eldoc elpy-modules)
```

Comment:
Shouldn't this always be the ase here?

Reply:
Yes, you are right, I meant `elpy-module-company`.
let me change that.